packages/stylable/src/stylable-meta.ts

```diff
@@ -53,18 +53,14 @@ export interface Imported {
     from: string;
     defaultExport: string;
     named: Pojo<string>;
-    overrides: postcss.Declaration[];
-    theme: boolean;
     rule: postcss.Rule;
     fromRelative: string;
 }
 
 export interface StylableDirectives {
     '-st-root'?: boolean;
-    '-st-compose'?: Array<ImportSymbol | ClassSymbol>;
     '-st-states'?: any;
     '-st-extends'?: ImportSymbol | ClassSymbol | ElementSymbol;
-    '-st-theme'?: boolean;
```

Comment:
need to check that intelligence won't break...

Reply:
Either way it'll be a major change and we'll have to go through the intelligence. Don't think this should be a big issue.